docs/intelligentapps/bulkrun.md

```diff
@@ -0,0 +1,42 @@
+---
+Order: 4
+Area: intelligentapps
+TOCTitle: Bulk Run
+ContentId:
+PageTitle: Bulk Run Prompts
+DateApproved:
+MetaDescription: Run a set of prompts in an imported dataset, individually or in a full batch towards the selected genAI models and parameters.
+MetaSocialImage:
+---
+
+# Bulk Run Overview
```

Comment:
Instead of using <feature name> overview, describe what it does: "Run multiple prompts in bulk"